study.md

```diff
@@ -160,15 +163,16 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+A response is the requested resource, and some important information relating
+to the request and resource, aka HTTP headers.
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+by entering this in the terminal "curl https://www.reddit.com/"
```

Comment:
It is best to be explicit so you will use the request flag and include the type of request like so:

`curl --request GET https://www.reddit.com`